Source/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -65,65 +65,75 @@ - (BOOL)presentExternalUserAgentRequest:(id<OIDExternalUserAgentRequest>)request
   NSURL *requestURL = [request externalUserAgentRequestURL];
 
   // iOS 12 and later, use ASWebAuthenticationSession
+  // ASWebAuthenticationSession doesn't work with guided access (rdar://40809553)
   if (@available(iOS 12.0, *)) {
```

Comment:
I would just add the extra condition here, i.e. `&& !UIAccessibilityIsGuidedAccessEnabled()` makes for less indentation.

Reply:
Actually Xcode gives a compilation error when chaining conditions with `@available`, see https://stackoverflow.com/questions/46965347/objective-c-available-guard-anded-with-more-conditions.